This test requires that '/sys/dev/block' is present, which is not
the case inside the build container.

--- a/tests/ts/lsblk/lsblk	2019-03-11 21:08:23.734387391 +0100
+++ b/tests/ts/lsblk/lsblk	2019-03-11 21:12:43.635473111 +0100
@@ -22,6 +22,8 @@
 
 ts_init "$*"
 
+ts_skip "/sys/dev/block is not available"
+
 ts_check_test_command "$TS_CMD_LSBLK"
 ts_check_prog xz
 ts_check_prog tar
